### PR TITLE
Update network direction enum to Include Lateral

### PR DESCRIPTION
**Pull request by Aniak5**
image

image

### CODE REVIEW

File: dictionary.json [hunk 1]
```diff
@@ -1078,6 +1082,9 @@
       "caption": "Direction ID",
       "description": "The direction of the initiated connection, traffic, or email.",
       "enum": {
```
Comment (Aniak5) [inline dictionary.json:1084]: Do we need this generic `direction_id` ? We only use it in the `network_connection` object and we redefine the enum values!

File: objects/network_connection.json [hunk 2]
```diff
@@ -22,6 +27,10 @@
         "2": {
           "description": "Outbound network connection. The connection was originated from inside the network, destined for services on the Internet or outside network.",
           "caption": "Outbound"
+        },
+        "3": {
+          "description": "Lateral network connection. The connection was originated from inside the network, destined for services on the inside network.",
```
Comment (Aniak5) [inline objects/network_connection.json:32]: For the `Lateral` enum I specifically called this out to be internal to internal bc moving laterally external to external doesn't really make sense to me but please let me know if y'all don't like this.

Comment (mr-splunk) [reply objects/network_connection.json:32]: Looks great

**Review by mr-splunk — APPROVED** [review]
Looks great.

**Pull request merged**